src/server/common.cc

```diff
@@ -186,6 +186,8 @@ TieredStats& TieredStats::operator+=(const TieredStats& o) {
   ADD(cold_storage_bytes);
   ADD(total_offloading_steps);
   ADD(total_offloading_stashes);
+
+  ADD(throttled_clients);
```

Comment:
From my past experience I find such metrics widely unstable as they probe behavior at this moment.
If say we probe metrics at intervals of 5s we could measure 0 at both probes while during the interval have moments with 100 clients being stalled. 

The more stable approach is to use aggregated events and then use `rate` functions to take derivatives